src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -22,8 +22,9 @@ import { AppState } from '../../store/app.reducers';
 
 import * as fromActions from './interactive-map.actions';
 import { environment } from '../../../environments/environment.staging';
-import { Cobra, CardType, MapItem } from '../types';
+import {Cobra, CardType, MapItem, Specie} from '../types';
 import { PathwayMap } from '@dd-decaf/escher';
+import {SetSelectedSpecies} from './interactive-map.actions';
```

Comment:
Actions are already available please use that import https://github.com/DD-DeCaF/caffeine/pull/40/files#diff-14cc7bc6ba94d28e4ada68d0f17157eeR23